### PR TITLE
Create models

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: app/models/user.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
+    type = Column(Enum(AccountType), default=AccountType.USER)
+    date = Column(DateTime, default=datetime.now())
+    username = Column(String(20), unique=True, index=True)
+    visible_name = Column(String(40))
+    desc = Column(String(400))
+    email = Column(String(50), unique=True, index=True)
+    password = Column(String)
```
Comment (jaanonim) [inline app/models/user.py:20]: limit size

**Review by jaanonim — APPROVED** [review]
LGTM

File: app/database.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./test.db"
+# SQLALCHEMY_DATABASE_URL = "postgresql://user:password@postgresserver/db"
```
Comment (Wuzado) [inline app/database.py]: Perhaps keep DB URL in the env vars, defaulting to SQLite? I think this way the production configuration will be much easier.

File: app/database.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+engine = create_engine(
+    SQLALCHEMY_DATABASE_URL, connect_args={"check_same_thread": False}
+)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+Base = declarative_base()
```
Comment (Wuzado) [inline app/database.py]: Add newline.

File: app/models/images.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+
+from ..database import Base
+
+
+class Images(Base):
+    __tablename__ = "images"
+
+    id = Column(Integer, primary_key=True, index=True)
+    url = Column(String(100))
```
Comment (Wuzado) [inline app/models/images.py]: Newline.

File: requirements.txt [hunk 5]
```diff
@@ -6,6 +6,7 @@ promise==2.3
 pydantic==1.7.3
 Rx==3.1.1
 six==1.15.0
-starlette==0.14.1
+starlette==0.13.6
```
Comment (Wuzado) [inline requirements.txt:9]: What's the reason for the version downgrade?

**Review by Wuzado — COMMENTED** [review]
A few nitpicks.

File: app/models/user.py (hunk 1 above)
Comment (n2oneProgrammer) [reply app/models/user.py:20]: it doesnt required, because hash always has same length

File: app/database.py (hunk 3 above)
Comment (n2oneProgrammer) [reply app/database.py]: how does it mean?

File: app/models/images.py (hunk 4 above)
Comment (n2oneProgrammer) [reply app/models/images.py]: how does it mean?

File: requirements.txt (hunk 5 above)
Comment (n2oneProgrammer) [reply requirements.txt:9]: fast API doesnt support starlette 0.14.1

File: app/models/user.py (hunk 1 above)
Comment (Wuzado) [reply app/models/user.py:20]: I believe that regards the limit of the password input.

**Review by Wuzado — APPROVED** [review]
LGTM

**Pull request merged**